substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ReachabilityHandlerFeature.java

```diff
@@ -119,7 +122,7 @@ public void duringAnalysis(DuringAnalysisAccess a) {
                 Set<Object> triggers = activeHandlers.get(callback);
                 if (callback instanceof Consumer) {
                     if (isTriggered(access, triggers)) {
-                        triggeredHandlers.put(callback, null);
+                        triggeredHandlers.put(callback, NULL_MAP);
```

Comment:
Why not just use `Map.of()`? We want that map to be immutable anyways.

Reply:
I didn't think about it. It makes much more sense. Done, thanks.